docs/README.md

```diff
@@ -0,0 +1,19 @@
+# US Forms System Documentation
+
+- [Getting started with the Forms System Library](getting-started/README.md)
+  - Creating a new application with the US Forms System Starter App
+  - Installing the library in an existing application
+- [Building a form](building-a-form/README.md)
+  - About the JSON schema standard and react-jsonschema-form
+  - Adding widgets
+  - Adding widget definitions
+  - Creating custom widgets
+  - Creating multi-page forms
+- [Form recipes](form-recipes/README.md)
```

Comment:
Do you think "recipes" is the best terminology to use here? Or should we just saw "common patterns", "common questions", "how tos" or something like that?

Reply:
I was mostly going off of what was in https://github.com/usds/us-forms-system/blob/master/docs/How%20the%20library%20works/form-config.md#schemaform-cookbook. I much prefer "common patterns" myself (or something similar) but didn't want to stray too far from the "cookbook" metaphor in case that was a beloved term. Glad to hear we're on the same page.